src/implementations/vanilla/components/hero-banner/_hero-banner.scss

```diff
@@ -36,6 +36,23 @@ $_font-weight: null !default;
   position: relative;
 }
 
+.ecl-hero-banner__copyright {
```

Comment:
it's true that we only use this twice, but couldn't we make a mixin and include it where needed, instead of duplicating these rules..?

Reply:
we are already duplicating all the other rules for the banners.
We could have some generic banner css, and just change what correspond to page or hero banner. But I would keep this for another ticket